cypress/e2e/setup-scripts/seed_superuser.py

```diff
@@ -56,7 +56,7 @@
         "access": "private",
     }
 
-    form = SLUG_MODEL_FORM_MAP.get(app_slug, None).Form(data, project_pk=project.pk)
+    form = SLUG_MODEL_FORM_MAP.get(app_slug, None).Form(data, project_pk=project.pk)  # type: ignore
```

Comment:
Why is this being catched?

Reply:
There was a mypy error: cypress/e2e/setup-scripts/seed_superuser.py:59: error: Item "None" of "ModelFormTuple | None" has no attribute "Form"  [union-attr]